src/client.rs

```diff
@@ -82,6 +89,7 @@ impl KaspadHandler {
     async fn handle_message(&mut self, msg: Payload, miner: &mut MinerManager) -> Result<(), Error> {
         match msg {
             Payload::BlockAddedNotification(_) => self.client_get_block_template().await?,
```

Comment:
We don't register to this notification now, so I would remove this match case

Reply:
Fixed, and also removed the corresponding `NotifyBlockAddedResponse`